### ISSUE TITLE
How can we train own model and annotations for this?

### ISSUE THREAD

**Issue by sharon-samson** [switchablenorms/DeepFashion2#69]
Hi Iam currently working on a research project, and it will be helpful if some one explain how can we train our own model and how to create its annotation files..

**Comment by HamzaZwairy** [comment]
@sharon-samson Hi, I don't know if I'm too late or not but I think that this tutorial in PyTorch helpful. Just create your own dataset class and go on with the tutorial, it applies Mask-RCNN which should do a decent job in an instance segmentation task. 

https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html

**Comment by sharon-samson** [comment]
I got segmented output. How can I make landmark points on it?
